index.html

```diff
@@ -0,0 +1,1011 @@
+                        </li>
+                        <li class="categories-li">
+                            Lynk & Co.'s car sharing-owning scheme will never cost more than €500 a month
+                        </li>
+                        <li class="categories-li">
+                            Lucid unveils the Air, its luxury EV with up to 517-miles of range costing as much as $169K
+                        </li>
+                    </ul>
+                </div>
+            </div>
+
+            <div class="categories-content">
+                <div class="categories-title">
+                    <h3>Distract<i class="fas fa-chevron-right"></i></h3>
+                </div>
+                <div class="categories-img ct-img-9">
+                    <div class="categories-img-txt">
+                        <h4>MOST POPULAR</h4>
+                        <a href="#" class="ct-img-txt-main">
+                            Amplitude celebrates 10 years with a free weekend of Endless games
+                        </a>
+                        <p class="ct-img-txt-sub">
+                            <a href="#">
+                                Tristan Greene</a> - 4 days ago
+                        </p>
+                    </div>
+                </div>
+                <div class="categories-subtitle-img">
+                    <div class="subtitle-left">
+                        <p class="subtitle-tile">LATEST</p>
+                        <p class="subtitle-txt">
+                            The Best vampire games and mods to play until Bloodlines 2 releases later this year
+                        </p>
+                    </div>
+                    <div class="subtitle-right ct-sub9">
+                    </div>
+                </div>
+                <div class="categories-links">
+                    <ul class="categories-ul">
+                        <li class="categories-li">
+                            Games to play on date night: Rule the galaxy together in Stellaris
+                        </li>
+                        <li class="categories-li">
+                            We need to make remote meetings more boring in 2021
+                        </li>
+                        <li class="categories-li">
+                            The podcasts that got us through 2020
+                        </li>
+                        <li class="categories-li">
+                            How Netflix shapes mainstream culture, explained by data
+                        </li>
+                        <li class="categories-li">
+                            Christmas is better with Tom Vasel’s YouTube board game reviews
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+
+        <!-- Latest deals -->
+        <div class="latest-deals-container">
+            <div>
+                <h3 class="ld-title">Latest deals <i class="fas fa-chevron-right"></i></h3>
```

Comment:
The title of this section should be clickable i.e. a link (`a` tag). I'd simply convert the surrounding `div` into an `a` tag.